src/Chartjs/assets/package.json

```diff
@@ -7,22 +7,30 @@
     "types": "dist/controller.d.ts",
     "symfony": {
         "controllers": {
-            "chart": {
-                "main": "dist/controller.js",
+            "chart_v3": {
+                "main": "dist/chart_v3_controller.js",
+                "name": "symfony--ux-chartjs--chart",
                 "webpackMode": "eager",
                 "fetch": "eager",
                 "enabled": true
+            },
+            "chart_v4": {
+                "main": "dist/chart_v4_controller.js",
+                "name": "symfony--ux-chartjs--chart",
+                "webpackMode": "eager",
+                "fetch": "eager",
+                "enabled": false
             }
         }
     },
     "peerDependencies": {
         "@hotwired/stimulus": "^3.0.0",
-        "chart.js": "^3.4.1"
+        "chart.js": "^3.4.1 <3.9 || ^4.0"
     },
     "devDependencies": {
         "@hotwired/stimulus": "^3.0.0",
         "@types/chart.js": "^2.9.34",
-        "chart.js": "^3.4.1 <3.9",
+        "chart.js": "^4.0",
```

Comment:
Make this look like `peerDependencies`. We don't do it at the moment, but we might/should add something to our CI to test "low" dependency versions as well as high (we do this in webpack-encore).